docs/reference/api-v1alpha.md

```diff
@@ -916,4 +916,540 @@ curl -i \
+  "created_at": "2023-06-01T10:07:36.000000Z",
+  "bookmark_parameter3": "",
+  "bookmark_parameter2": "",
+  "bookmark_parameter1": "",
+  "active": true
+}
+```
+
+**Example**
+
+```
+curl -i -H "Authorization: Token {api_token}" \
+     "https://{org_name}.semaphoreci.com/api/v1alpha/deployment_targets/:target_id"
+```
+
+#### Describing by name and project UUID
+
+This API endpoint retrieves a deployment target based on its name and the UUID of the project it is linked to.
+
+```
+GET {org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id&target_name=:target_name
+```
+
+**Params**
+
+- `project_id` (**required**) - The UUID of the project to which the deployment target is linked.
+- `target_name` (**required**) - The name of the deployment target.
+
+**Response**
+
+```json
+HTTP status: 200
+
+[
+  {
+    "url": "project_1234.zyx",
+    "updated_by": "02984c87-efe8-4ea1-bcac-9511a34a3df3",
+    "updated_at": "2023-06-07T09:24:04.000000Z",
+    "subject_rules": [
+      {
+        "type": "ANY",
+        "subject_id": ""
+      }
+    ],
+    "state_message": "",
+    "state": "USABLE",
+    "project_id": "a426b4db-1919-483d-926d-06ba1320b209",
+    "organization_id": "7304b7f9-7482-46d4-9b95-3cd5a6ef2e6f",
+    "object_rules": [
+      {
+        "type": "BRANCH",
+        "pattern": "",
+        "match_mode": "ALL"
+      },
+      {
+        "type": "TAG",
+        "pattern": "",
+        "match_mode": "ALL"
+      },
+      {
+        "type": "PR",
+        "pattern": "",
+        "match_mode": "ALL"
+      }
+    ],
+    "name": "DTName123",
+    "last_deployment": null,
+    "id": "e79d3d1c-61cc-4e07-ba32-f86694f5e80d",
+    "description": "DT description",
+    "created_by": "02984c87-efe8-4ea1-bcac-9511a34a3df3",
+    "created_at": "2023-06-01T10:07:36.000000Z",
+    "bookmark_parameter3": "",
+    "bookmark_parameter2": "",
+    "bookmark_parameter1": "",
+    "active": true
+  }
+]
+```
+
+**Example**
+
+```
+curl -i -H "Authorization: Token {api_token}" \
+     "https://{org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id&target_name=:target_name"
+```
+
+### Creating a target
+
+This API endpoint allows you to create a new deployment target and assign it to a specific project.
+
+```
+POST {org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id
+```
+
+**Request Body**
+
+The request body should be a JSON object, encapsulating details about the deployment target to be created. The available fields are as follows:
+
+  - `name` (**required**) - Unique name for the target within the project.
+  - `project_id` (**required**) - UUID of the project.
+  - `unique_token` (**required**) - Idempotency UUID token.
+  - `description` (*optional*) - A description of the target.
+  - `url` (*optional*) - The URL of the target.
+  - `bookmark_parameter1`, `bookmark_parameter2`, `bookmark_parameter3` (*optional*) - Bookmark parameters.
```

Comment:
Let's expand the description of what bookmarks are here. You can reuse some parts from the CLI docs.